vocabularyEntryDetails.md

```diff
@@ -0,0 +1,167 @@
+# Vocabulary Entries Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+All the endpoints here described are restricted to authenticated users.
+
+## Main Endpoint
+**/api/submission/vocabularyEntryDetails**   
+
+Not allowed. Only subset of vocabulary entries can be retrieved using specific filters see below
+
+## Single Authority
+**/api/submission/vocabularyEntryDetails/<vocabulary-name>:<entry-id>**
```

Comment:
I assume this implies the vocabulary-name cannot contain a colon?

Reply:
you are right. I added a note in the id/name attribute description

Reply:
Thanks, I just noticed there's a problem with how this is rendered, visible at https://github.com/4Science/Rest7Contract/blob/controlled_vocabularies/vocabularyEntryDetails.md#single-authority
This should be solved by using `<:vocabulary-name>:<:entry-id>`

Reply:
fixed, thanks